examples/python-api/myownbloom.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+import khmer
+
+d1="ATGTACGGGCATTACGATTACCGATGTAG";
+d2="ATGACCAAACTCATTACGATTAGATATAG";
+
+ksize = 5
+target_table_size = 5e8
+num_tables = 4
+
+bf =khmer.Nodetable(ksize, target_table_size, num_tables);
+bf.consume(d1);
+
+cms=khmer.Counttable(ksize, target_table_size, num_tables);
+
+for kmer in cms.get_kmers(d2): #each kmer in d2
+	if bf.get(kmer) == 0:#if not in d1=
+		cms.consume(kmer)
+assert cms.get('CCAAA') 
+
+assert bf.get('ACCGA') 
```

Comment:
This command is testing whether the k-mer is in the mask. I'm not sure that's a helpful test.

What *would* be helpful is to find a k-mer that is present in both d1 and d2 and make sure it's NOT in cms (that is `assert cms.get(thekmer) == 0`).